pkg/virtual/framework/dynamic/apiserver/discovery.go

```diff
@@ -107,29 +108,25 @@ func (r *versionDiscoveryHandler) ServeHTTP(w http.ResponseWriter, req *http.Req
 		apiResourceSpec := apiDef.GetAPIResourceSpec()
 		subresources = apiResourceSpec.SubResources
 
-		// TODO: get the list of verbs from the REST storage instance
-		verbs := metav1.Verbs([]string{"get", "list", "patch", "create", "update", "watch"})
```

Comment:
... why did this not include delete ?

Reply:
Because it was initially written for the Syncer virtual workspace, where we never intent to delete anything on the kcp logical cluster.